c2cgeoportal/scaffolds/create/setup.cfg_tmpl

```diff
@@ -4,3 +4,7 @@ nocapture = 1
 cover-package = {{package}}
 with-coverage = 1
 cover-erase = 1
+
+[flake8]
+ignore = E121,E123,E126,E226,E24,E704,E302,E265
```

Comment:
I prefer to have by default the standard list: `E121,E123,E126,E226,E24,E704` :-)